libs/common/rendersettings_utils.cpp

```diff
@@ -475,7 +476,7 @@ void ReadRenderSettings(const UsdPrim &renderSettingsPrim, ArnoldAPIAdapter &con
             continue;
 
         AtNode *driver = nullptr;
-        if (productPrim.HasAttribute(TfToken("arnold:driver"))) {
+        if (HasAuthoredAttribute(productPrim, _tokens->aovDriver)) {
```

Comment:
but we don't add arnold:driver to the RenderProduct schema ? do we ?

Reply:
not at the moment. But since the test we want to do here is actually to know if the attribute is authored, I think it makes more sense to use the new explicit method. At least if we add it to the schemas in the future we won't have the problem